docs/public/README.md

```diff
@@ -656,17 +650,39 @@ The application plane emits this event upon completion of tenant deactivation. A
+
+Many SaaS providers rely on integration with a billing service that allows them to created the billing plans that are mapped to their tiering and monetization strategy. Generally, the approach here is to provide an API within the control plane that will provide a universal mechanism for creating billing accounts (onboarding) and publishing billing/consumption events for individual tenants. This service will also include integration with tenant management, enabling tenant state events to be conveyed to/from the billing service. For example, a tenant being disabled could be triggered from billing to tenant management. A change in tiers could also be part of this integration.
 
 #### Metrics
+
 Multi-tenant operations teams often need tenant-aware insights into the activity and consumption profiles of their tenants and tiers. This data is used in a variety contexts, allowing teams to make operational, business, and design decisions based on the tenant-aware patterns they can observe through metrics that are published by the application tier. The control plane will provide standardized metrics API that will be used to ingest these metrics events. It will also enable builders to define custom events with some standardization around how tenant and tier context is conveyed via these events. These metrics will all be aggregated by the control plane and surfaced through dashboards in the control plan admin console. There may also be support for ingesting the metrics data into other analytics tools
+
 #### Tiering
-Tiering is a very basic construct in the control plane. However it needs to exist to provider a universal, centralized home to the tiering configuration of a SaaS environment. This provides a point of alignment between the application plane and the control plane, that allows for a clear mapping of tenants to tiers that are used across a number of experiences insides and outside of the control plane.  
+
+Tiering is a very basic construct in the control plane. However it needs to exist to provider a universal, centralized home to the tiering configuration of a SaaS environment. This provides a point of alignment between the application plane and the control plane, that allows for a clear mapping of tenants to tiers that are used across a number of experiences insides and outside of the control plane.
+
+### AWS Marketplace Integration
+
```

Comment:
Given, we expect more builders to use this marketplace component, I think we need more detailed readme for Marketplace Integration regarding how to use and what are different components..etc. 
While going through the code I could not related to what it is for..etc.

Maybe your video and bit more documentation might help.

Reply:
once this is merged, construct hub will have a better view of low level details of setting this construct up

Reply:
on the actual documentation, i think we can borrow a lot of artifacts from the marketplace quickstart since the two solutions are very similar.

Reply:
Added more docs: https://github.com/awslabs/sbt-aws/blob/13d4f239ac9784cef5ee473cdf65fc4dadd57389/docs/public/marketplace-integration.md